piksi_tools/udp_bridge.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Fergus Noble <fergus@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+the :mod:`sbp.client.examples.mavproxybridge` module contains an example of reading SBP
+messages from a serial port, filtering for observations, and sending them over udp
+to a mavproxy instance for transmission to an ArduCopter quad.
+"""
+
+from sbp.client.drivers.pyserial_driver import PySerialDriver
+from sbp.client.handler import Handler
+from sbp.observation import SBP_MSG_OBS, SBP_MSG_BASE_POS
+
+import socket
+import struct
+import time
+
```

Comment:
An integration test for this stuff would be cool. I.e., some way of reading a message log, serializing, sending it over UDP and then catching it on the other side. 

Here's an example of how to setup a mock socket server: https://github.com/swift-nav/pySTR4500/blob/master/test/test_client.py. 

You can also construct objects inline, `.pack()`, and dispatch over UDP. For inline construction, see: https://github.com/swift-nav/sbp_log_analysis/blob/master/tests/test_rtk_fix_loss.py#L29